### PR TITLE
Vectorized NormalizedError

### PR DESCRIPTION
**Pull request by K20shores**
Create a vectorized version of the normalized error function

### CODE REVIEW

File: include/micm/solver/rosenbrock.inl [hunk 1]
```diff
@@ -581,4 +588,56 @@ namespace micm
...
+
+    // compute the error over the remaining elements that are in the next group but didn't fill a full group
+    size_t remaining_blocks = Y.NumRows() % Y.GroupVectorSize();
+    const size_t L = Y.GroupVectorSize();
+    
+    if (remaining_blocks > 0){
+      for(std::size_t y = 0; y < Y.NumColumns(); ++y) {
+        for(std::size_t x = 0; x < remaining_blocks; ++x) {
+          size_t idx = y * L + x;
+          errors_over_scale = errors_iter[idx] / (parameters_.absolute_tolerance_[idx % n_species] +
```
Comment (K20shores) [inline include/micm/solver/rosenbrock.inl:632]:
```suggestion
          errors_over_scale = errors_iter[idx] / (parameters_.absolute_tolerance_[y % n_species] +
```

should it actually be this?

File: include/micm/solver/rosenbrock.inl [hunk 2]
```diff
@@ -581,4 +588,56 @@ namespace micm
...
+    }
+
+    // compute the error over the remaining elements that are in the next group but didn't fill a full group
+    size_t remaining_blocks = Y.NumRows() % Y.GroupVectorSize();
+    const size_t L = Y.GroupVectorSize();
+    
+    if (remaining_blocks > 0){
+      for(std::size_t y = 0; y < Y.NumColumns(); ++y) {
+        for(std::size_t x = 0; x < remaining_blocks; ++x) {
+          size_t idx = y * L + x;
```
[the remark this answers is not in the document]
Comment (sjsprecious) [reply include/micm/solver/rosenbrock.inl:631]: Never mind. I am wrong and the current implementation is correct.

**Review by sjsprecious — APPROVED** [review]
@K20shores Nice work and thanks for addressing this issue!

**Review by boulderdaze — CHANGES_REQUESTED** [review]
Looks good! Just a couple comments to enhance the code consistency within the same function

**Review by boulderdaze — APPROVED** [review]
Thanks!

**Review by mattldawson — APPROVED** [review]
looks great!

**Pull request merged**